chalk/shapes/__init__.py

```diff
@@ -89,11 +89,11 @@ def rectangle(
     Returns:
         Diagrams
     """
-    return Primitive.from_shape(Rectangle(width, height, radius))
+    return Path.rectangle(width, height).stroke()
```

Comment:
The `radius` argument is not used.

Reply:
Ah yeah. Maybe I will do this as a second PR. I want to make sure the rounding is the same in each of the different backends.